types/react-relay/classic.d.ts

```diff
@@ -209,13 +209,17 @@ export type OnReadyStateChange = (
     }
 ) => void;
 
-export interface RelayProp {
+export interface RelayProp<V> {
```

Comment:
Can you make this default to `any` so it stays backwards compatible?

Reply:
Indeed, good idea.

Reply:
That done.